src/bot.py

```diff
@@ -156,12 +157,30 @@ def lastgame(update: Update, context: CallbackContext):
                                      text=time,
                                      parse_mode=ParseMode.HTML)
 
+def birthdays(update: Update, context: CallbackContext):
```

Comment:
Seems a bit wastefull to load the Birthdays everytime we issue a command, perhaps I could be more efficient to have a globally persistent object for them. This is a small niggle and more trouble than it would be worth to us I suppose but maybe something to keep in mind for the future in this project ^^